ygot/diff.go

```diff
@@ -488,6 +489,41 @@ func Diff(original, modified GoStruct, opts ...DiffOpt) (*gnmipb.Notification, e
 	}
 }
 
+// FormatDiff formats the output of ygot.Diff as a multiline string. This
+// function is only intended for human consumption and ignores errors. Do not
+// depend on the output being stable. It may change over time across different
+// versions of the program.
+func FormatDiff(n *gnmipb.Notification) string {
```

Comment:
I wonder whether this should go in `testutil` -- given that it seems like it might be a testing-only function, or do we expect production code to maybe use this for errors?

Reply:
I think this makes it more discoverable since autocompleters might pick it up (mine does). Otherwise it would be hidden. I'm using the pattern from prototext.Format, which lives alongside prototext.Marshal/Unmarshal.

Reply:
SGTM -- I agree with the discoverability argument.